src/test/java/org/prebid/server/bidder/nextmillennium/NextMillenniumBidderTest.java

```diff
@@ -54,6 +57,62 @@ public void setUp() {
         nextMillenniumBidder = new NextMillenniumBidder(ENDPOINT_URL, jacksonMapper);
     }
 
+    @Test
+    public void makeHttpRequestsBidRequestShouldHaveIdenticalParams() {
+        ExtImpNextMillennium initialExt = ExtImpNextMillennium.of("6821", null);
+        Imp initialImp = givenImpWithExt(imp -> imp
+                        .id("custom_imp_id")
+                        .tagid("custom_imp_tagid")
+                        .secure(1)
+                        .banner(Banner.builder()
+                                .w(728)
+                                .h(90)
+                                .format(singletonList(Format.builder()
+                                        .w(728)
+                                        .h(90)
+                                        .build())).build()
+                        ),
+                initialExt
+        );
+        BidRequest bidRequest = givenBidRequest(b -> b
+                .id("c868fd0b-960c-4f49-a8d6-2b3e938b41f2")
+                .test(1)
+                .tmax(845L)
+                .device(Device.builder().w(994).h(1768).build())
+                .imp(singletonList(initialImp))
+                .ext(ExtRequest.of(ExtRequestPrebid.builder()
+                        .schains(emptyList())
+                        .targeting(ExtRequestTargeting.builder()
+                                .includewinners(true)
+                                .includebidderkeys(false)
+                                .build())
+                        .build()))
+                .site(Site.builder().page("https://www.advfn.com").domain("www.advfn.com").build())
+        );
+
+        final Result<List<HttpRequest<BidRequest>>> result = nextMillenniumBidder.makeHttpRequests(bidRequest);
+        Optional<BidRequest> searchResult = result.getValue()
+                .stream()
+                .map(HttpRequest::getPayload)
+                .findFirst();
+        BidRequest modifiedRequest = searchResult.get();
+        Imp modifiedImp = modifiedRequest.getImp().get(0);
+        // Bid Request
+        assertThat(modifiedRequest)
```

Comment:
We prefer to test different objects in different unit tests.
Pls make a separate test for imp and bidRequest